hack/manage-mocks.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+COPYRIGHT_FILE=./hack/boilerplate/boilerplate.generatego.txt
+TARGET_DIR=./pkg/azureclients
+IFACE_FILE=interface.go
```

Comment:
Could you read these two params from environment varilable?

Reply:
can you please point to any example which will help me resolve this comment.

Also Need help with unit test debugging...I checked the generated mock_client...only boilerplate part is updated..remaining file is same..pointers regarding the same will also be helpful

Reply:
Thanks for working on this. I pushed the required changes to get this merged sooner.